ksql-common/src/main/java/io/confluent/ksql/schema/ksql/Column.java

```diff
@@ -29,9 +29,9 @@
 public final class Column implements SimpleColumn {
 
   public enum Namespace {
-    META,
     KEY,
-    VALUE
+    VALUE,
```

Comment:
please add a comment explaining that the order of the enum is important (re `LogicalSchema`) - even though this is covered in tests it's nice to know that when reading the code

Reply:
It won't be needed for long, but OK